src/auth/reset_password/reset_password.html

```diff
@@ -0,0 +1,19 @@
+<form id="auth.reset_password.form" name="form" class="inside form-horizontal" method="post" ng-submit="send()">
+    <div class="well3">
```

Comment:
This div is probably unnecessary and we don't use or style such class anywhere. Please remove it.